test/results/muxer.parsed

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
I can't comment on the binaries itself but all that should be put in `test/fixtures` folder, not `test/results`

Reply:
And please change the extension name to `.bin` or something like that so github does not treat it as text files, then it won't show 750k of changed lines.